ast/check.go

```diff
@@ -696,7 +695,7 @@ func (rc *refChecker) checkRef(curr *TypeEnv, node *typeTreeNode, ref Ref, idx i
 			rc.env.tree.PutOne(value, types.S)
 		}
 
-	case Ref:
+	case Ref: // TODO(sr): Can this happen? Aren't refs in refs rewritten to the Var case above?
```

Comment:
Agreed, this should not be possible against compiled modules because of rewriting. I think this path predates some of the rewriting implementation in the compiler.